lib/cdo/shared_constants.rb

```diff
@@ -790,4 +790,15 @@ module SharedConstants
   AICHAT_METRICS_NAMESPACE = 'GenAICurriculum'.freeze
 
   US_STATES = STATE_ABBR_WITH_DC_HASH.merge(DC: 'Washington, D.C.').sort_by(&:last).to_h.freeze
+
+  PROJECT_SUBMISSION_STATUS = {
```

Comment:
nit: should these just be strings instead of numeric codes? I think we used numbers for some other constants like the AI Chat execution status for example, so that the model field could be numeric. But it might be clearer to have these be strings if they're only being used for reference?